app/components/modal-delete-member.js

```diff
@@ -1,25 +1,46 @@
 import ModalComponent from 'ghost-admin/components/modal-base';
 import {alias} from '@ember/object/computed';
+import {computed} from '@ember/object';
 import {inject as service} from '@ember/service';
 import {task} from 'ember-concurrency';
 
 export default ModalComponent.extend({
     membersStats: service(),
 
+    cancelSubscriptions: false,
```

Comment:
Can we change this to `shouldCancelSubscriptions`? I was tripped up thinking we were passing a function/action around when reading later code 😄

Reply:
Yes! Good point :relaxed: